src/main/java/redis/clients/jedis/BinaryShardedJedis.java

```diff
@@ -1124,14 +1124,15 @@ public Long xtrim(byte[] key, long maxLen, boolean approximateLength) {
   }
 
   @Override
-  public List<byte[]> xpending(byte[] key, byte[] groupname, byte[] start, byte[] end, int count, byte[] consumername) {
+  public List<Object> xpending(byte[] key, byte[] groupname, byte[] start, byte[] end, int count,
```

Comment:
Any reason to not use `List<byte[]>` here?

Reply:
From [Redis doc](https://redis.io/commands/xpending):
```
> XPENDING mystream group55
1) (integer) 1
2) 1526984818136-0
3) 1526984818136-0
4) 1) 1) "consumer-123"
      2) "1"
```
The response can't be represented by `List<byte[]>`.